install.rst

```diff
@@ -0,0 +1,110 @@
+- `OMERO installation workshop presentation <https://downloads.openmicroscopy.org/presentations/2020/Dundee/Workshops/OME2020-OMERO-Installation/#/>`_
+- `Ansible <https://www.ansible.com/>`_
+- `Ansible documentation <https://docs.ansible.com/ansible_community.html>`_
+- Link to install docs
+- Link to requrements
+- Link to playbooks
+- Link to App list
+
+Install a server
+----------------
+
+Get an empty box or virtual machine with CentOS 7.
+
+Ansible
+-------
+
+For OMERO installation, we recommend to use the Ansible software suite https://docs.ansible.com/ansible/latest/index.html which enables infrastructure as a code.
+
+Install ansible (at the time of writing, the ansible-core version was 2.11.12, python version was 3.6.8). Other versions might work too, but are not routinely tested.::
+
+    $ pip install ansible
+
+Clone the repository for the one-node example::
+
+    $ git clone https://github.com/ome/ansible-example-omero-onenode.git
+    $ cd ansible-example-omero-onenode	
+
+Install requrements::
+
+    $ ansible-galaxy install -r requirements.yml
+
+Edit the file ``hosts.yml`` in the same directory and replace the YOUR-HOST-NAME variable with a name of your CentOS 7 box you prepared in the step above to install OMERO on, e.g. ``foo.example.com`` or ``localhost``)::
+
+    all:
+      hosts:
+        YOUR-HOST-NAME 
+
+Edit the ``playbook.yml`` file to change the OMERO root password (by default ``ChangeMe``). Then use this playbook to install PostgreSQL, OMERO.server and OMERO.web on one node::
+
+    $ ansible-playbook --become -i hosts.yml playbook.yml
+
+Start OMERO.insight or a Command Line Interface (CLI) and log in to ``YOUR-HOST-NAME`` with the username ``root`` and the password you have set in the ``playbook.yml`` above.
+
+Go to ``https://YOUR-HOST-NAME:4080/webclient/`` in your browser and log in to OMERO.web with the same credentials you used for OMERO.insight above. In case you are on the same machine where you installed your OMERO, go to `http://localhost:4080/webclient/ <http://localhost:4080/webclient/>`_ to access your OMERO.web locally.
+
+Docker
+------
+
+Even quicker option than ansible is docker, but you have to be comfortable working with containers. The docker workflow below is written for local installation of OMERO only for convenience, but bear in mind that you will have to install OMERO on a remote server when going into production.
+
+Assuming that Docker is installed, clone the deployment examples repository (if not already cloned)::
+
+    $ git clone https://github.com/ome/omero-deployment-examples.git
+    $ cd omero-deployment-examples
+    $ git clone https://github.com/ome/docker-example-omero.git
+    $ cd docker-example-omero
+
+Pull the latest versions of the containers and start them::
+
+    $ docker-compose pull
+    $ docker-compose up -d
+    $ docker-compose logs -f
```

Comment:
This works so far... and gets to:
```
omeroserver_1  | 2022-09-22 13:08:48,441 [    omego.db] INFO  Database is already at OMERO5.4__0
omeroserver_1  | Running /startup/99-run.sh 
omeroserver_1  | Starting OMERO.server
```
but then seems to stop.
Webclient is available at http://localhost:4080/webclient but says server is not responding.
How do I access server logs to see if anything went wrong?

Reply:
I have:
```
$ docker ps
CONTAINER ID   IMAGE                                   COMMAND                  CREATED      STATUS         PORTS                              NAMES
4e63dfd4de54   openmicroscopy/omero-server:5           "/usr/local/bin/entr…"   5 days ago   Up 8 minutes   0.0.0.0:4063-4064->4063-4064/tcp   docker-example-omero_omeroserver_1
7bb7d25fb5f7   openmicroscopy/omero-web-standalone:5   "/usr/local/bin/entr…"   5 days ago   Up 8 minutes   0.0.0.0:4080->4080/tcp             docker-example-omero_omeroweb_1
acb5f42fa4cf   postgres:11                             "docker-entrypoint.s…"   5 days ago   Up 8 minutes   5432/tcp                           docker-example-omero_database_1
```

Reply:
OK, so I managed to ssh into server...

```
$ docker exec -it docker-example-omero_omeroserver_1 /bin/bash
```
I don't know where `$OMERO_HOME` is, thought maybe I could find something like `grid.xml`...

```
bash-4.2$ cd /
bash-4.2$ find ./ -name grid.xml
find: './var/spool/at': Permission denied
find: './var/spool/cron': Permission denied
find: './var/db/sudo': Permission denied
find: './var/lib/machines': Permission denied
find: './var/lib/yum/history/2020-11-13/2': Permission denied
find: './var/lib/yum/history/2020-11-13/1': Permission denied
find: './var/lib/yum/history/2020-11-13/8': Permission denied
find: './var/lib/yum/history/2020-11-13/11': Permission denied
find: './var/lib/yum/history/2020-11-13/7': Permission denied
find: './var/lib/yum/history/2020-11-13/9': Permission denied
find: './var/lib/yum/history/2020-11-13/6': Permission denied
find: './var/lib/yum/history/2020-11-13/10': Permission denied
find: './var/lib/yum/history/2020-11-13/5': Permission denied
find: './var/lib/yum/history/2020-11-13/3': Permission denied
find: './var/lib/yum/history/2020-11-13/4': Permission denied
find: './etc/pki/CA/private': Permission denied
find: './etc/ssmtp': Permission denied
find: './etc/sudoers.d': Permission denied
find: './proc/tty/driver': Permission denied
find: './run/cryptsetup': Permission denied
find: './run/sudo': Permission denied
find: './root': Permission denied

bash-4.2$ sudo find ./ -name grid.xml
sudo: no tty present and no askpass program specified

bash-4.2$ echo $OMERO_HOME

```

Reply:
Thanks - using my `exec` still on omero-server container... This gives me nothing, so something failing before logs created:
```
ls /opt/omero/server/OMERO.server/var/
```

Reply:
> I have:
> 
> ```
> $ docker ps
> CONTAINER ID   IMAGE                                   COMMAND                  CREATED      STATUS         PORTS                              NAMES
> 4e63dfd4de54   openmicroscopy/omero-server:5           "/usr/local/bin/entr…"   5 days ago   Up 8 minutes   0.0.0.0:4063-4064->4063-4064/tcp   docker-example-omero_omeroserver_1
> 7bb7d25fb5f7   openmicroscopy/omero-web-standalone:5   "/usr/local/bin/entr…"   5 days ago   Up 8 minutes   0.0.0.0:4080->4080/tcp             docker-example-omero_omeroweb_1
> acb5f42fa4cf   postgres:11                             "docker-entrypoint.s…"   5 days ago   Up 8 minutes   5432/tcp                           docker-example-omero_database_1
> ```

vs mine which is  (I have some additional :: motives in the PORTS bit, but not sure how important this is)

```
docker ps
CONTAINER ID   IMAGE                                   COMMAND                  CREATED          STATUS          PORTS                                                           NAMES
3d049e9f93a0   postgres:11                             "docker-entrypoint.s…"   35 seconds ago   Up 34 seconds   5432/tcp                                                        docker-example-omero_database_1
8fd37b70d98b   openmicroscopy/omero-web-standalone:5   "/usr/local/bin/entr…"   35 seconds ago   Up 34 seconds   0.0.0.0:4080->4080/tcp, :::4080->4080/tcp                       docker-example-omero_omeroweb_1
3e3b742adb6d   openmicroscopy/omero-server:5           "/usr/local/bin/entr…"   35 seconds ago   Up 34 seconds   0.0.0.0:4063-4064->4063-4064/tcp, :::4063-4064->4063-4064/tcp   docker-example-omero_omeroserver_1
```

Reply:
@will-moore I have master.err and Blitz log, but my server is not working on M1. I will open an extra issue, so as not to confuse people.

Reply:
@will-moore created https://github.com/ome/docker-example-omero/issues/12